compiler/damlc/tests/src/DamlcVisual.hs

```diff
@@ -26,4 +29,12 @@ unitTests = do
                 dotFile
                 path
                 (execVisual darPath (Just path))
+            , testCase "multiline manifest file test" $
+                assertEqual "is this even running"
```

Comment:
That is an odd message in case of failure.

Reply:
:(